src/models/db.js

```diff
@@ -46,5 +69,17 @@ module.exports = config => {
   db.sequelize = sequelize;
   db.Sequelize = Sequelize;
 
-  return db;
+  return db.sequelize.sync().then(() => {
+    while (deferreds.length) {
+      deferreds.pop().resolve(db);
+    }
+    state = READY;
+
+    return db;
+  })
+  .catch(err => {
```

Comment:
I didn't realize what you meant with your comment about how the linter wants catches. If this is the way it likes, I prefer to change it :) It looks better in the previous line.